src/dinit.cc

```diff
@@ -486,6 +588,18 @@ int dinit_main(int argc, char **argv)
         }
     }
 
+#if SUPPORT_SELINUX
+    // Error exit if we are PID 1 and fail to load the selinux policy and transition.
+    //
+    // This should be done directly after argument parsing, it's best to do this as early as possible to get
```

Comment:
This comment line, in contrast, is too _long_. And so are some of the others. Please check all comments and make sure the line widths are correct.

Basically: they should be as close to 100 characters as is possible, and may extend to 110 in cases where that means the entire comment will fit on a single line.